config/env-template

```diff
@@ -0,0 +1,8 @@
+GRIST_DOMAIN=grist.localhost
+AUTHELIA_SUBDOMAIN=authelia
+DEFAULT_EMAIL=test@example.org
+PERSIST_DIR=./persist
+SECRETS_DIR=./config/secrets
+GRIST_CLIENT_SECRET=
+GRIST_OIDC_IDP_CLIENT_ID=
+GRIST_OIDC_IDP_CLIENT_SECRET=
```

Comment:
Currently need to be generated manually, should be generated by the scripts.

Reply:
Done now.